### PR TITLE
WFLY-15364 Consolidate IdentifierFactory interfaces/implementations across distributed web & ejb modules

### PR DESCRIPTION
**Pull request by pferraro**
https://issues.redhat.com/browse/WFLY-15364

### CODE REVIEW

**Comment by pferraro** [comment]
@rhusar @rachmatowicz Please review.

File: clustering/web/cache/src/main/java/org/wildfly/clustering/web/cache/sso/CompositeSSOManager.java [hunk 1]
```diff
@@ -22,10 +22,11 @@
 package org.wildfly.clustering.web.cache.sso;
```
Comment (rachmatowicz) [inline clustering/web/cache/src/main/java/org/wildfly/clustering/web/cache/sso/CompositeSSOManager.java:22]: A number of header files, as in this one, have old dates in them...

Comment (pferraro) [reply clustering/web/cache/src/main/java/org/wildfly/clustering/web/cache/sso/CompositeSSOManager.java:22]: The dates reflect the year they were created.

**Review by rachmatowicz — APPROVED** [review]
Looks good to me.

File: clustering/web/cache/src/main/java/org/wildfly/clustering/web/cache/sso/CompositeSSOManager.java (hunk 1 above)
Comment (rachmatowicz) [reply clustering/web/cache/src/main/java/org/wildfly/clustering/web/cache/sso/CompositeSSOManager.java:22]: So they are not updated when changes are made, OK.

**Pull request merged**